RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cfi.py

```diff
@@ -4,20 +4,39 @@
 
 #energy corrector for corrected cluster producer
 _emEnergyCorrector = cms.PSet(
-    algoName = cms.string("PFClusterEMEnergyCorrector"),
+    algoName = cms.string('PFClusterEMEnergyCorrector'),
     applyCrackCorrections = cms.bool(False),
     applyMVACorrections = cms.bool(True),
     maxPtForMVAEvaluation = cms.double(90.),
     recHitsEBLabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEB'),
     recHitsEELabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEE'),
+    ebSrFlagLabel = cms.InputTag('ecalDigis'),
+    eeSrFlagLabel = cms.InputTag('ecalDigis'),
     autoDetectBunchSpacing = cms.bool(True),
+    srfAwareCorrection = cms.bool(False)
+)
+
+_emEnergyCorrector_2017 = cms.PSet(
+    algoName = cms.string('PFClusterEMEnergyCorrector'),
+    applyCrackCorrections = cms.bool(False),
+    applyMVACorrections = cms.bool(True),
+    maxPtForMVAEvaluation = cms.double(300.),
+    recHitsEBLabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEB'),
+    recHitsEELabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEE'),
+    ebSrFlagLabel = cms.InputTag('ecalDigis'),
+    eeSrFlagLabel = cms.InputTag('ecalDigis'),
+    autoDetectBunchSpacing = cms.bool(True),
+    srfAwareCorrection = cms.bool(True)
 )
 
 particleFlowClusterECAL = cms.EDProducer(
-    "CorrectedECALPFClusterProducer",
-    inputECAL = cms.InputTag("particleFlowClusterECALUncorrected"),
-    inputPS = cms.InputTag("particleFlowClusterPS"),
+    'CorrectedECALPFClusterProducer',
+    inputECAL = cms.InputTag('particleFlowClusterECALUncorrected'),
+    inputPS = cms.InputTag('particleFlowClusterPS'),
     minimumPSEnergy = cms.double(0.0),
     energyCorrector = _emEnergyCorrector
     )
 
+
```

Comment:
Yes, I noticed too that HLT doesn't use this correction. I was wrong above. The HLT is using a very old parametrized correction. The reason for that, I do not understand.

Maybe @Sam-Harper knows why the same energy correction is not activated at HTL.